src/karmen_frontend/src/components/forms/form-utils.js

```diff
@@ -105,7 +107,34 @@ export const FormInputs = ({ definition, updateValue }) => {
         return null;
     }
   });
+};
 
+export const FormInputs = ({
+  definition,
+  updateValue,
+  collapsibleDefinition,
+}) => {
+  const optionRows = [
+    <InputSwitch definition={definition} updateValue={updateValue} />,
+  ];
+  if (collapsibleDefinition) {
+    optionRows.push(
+      <>
+        <span></span>
```

Comment:
What's the purpose of blank spans here and after the `Collapsible`?

Reply:
@tibor-kulcar ?

Reply:
It's about layout of form - 3 columns (label, input, error msg)

Reply:
@tibor-kulcar Can't we come up with some better approach? This hard-codes the form layout and will have to be reworked if used in different layout. Simply put, it expects things defined elsewhere making it easy to break.

Reply:
It is possible, but it isn't the scope of this issue. Please, create an issue on it.